analysis/images-cellcycleR.Rmd

```diff
@@ -0,0 +1,368 @@
+
+$I_{nj}$: Background-correctd intensity values in single cell sample $n$ and batch $j$.
+
+$\bar{I}$: Mean background-correctd intensity values across sample $n=1,\dots, N$ and batches $j=1,\dots,J$.
+
+$S$: Standard deviation of background-corrected intensity values $I_{nj}$. 
+
+
+$$
+Z = \frac{I_{nj} - \bar{I}}{S}
+$$
+
+$~$
+
+### Results
+
+1. Fit 200 interations. loglik = -3369.
+
+2. Fit appears to be better for RFP than for GFP and DAPI.
+
+3. Considering clustering of samples in the bivariate distribution of Red and Gree intensities. The sample clusters appear to coincide with the estimated cell times. 
+
+4. Consider residual intensities at the individual level, variation around the mean for all three channles is smaller than the observed intensity.
+
+5. Consider residual intensity at the batch level within each individual, standard deviations are smaller for Red than for Green and DAPI. Moreover, for the most part, means of the residuals center at the mean for all channels, and more so for Red than for Green across the batches. These results suggest that there's no need to fit a batch-level model for intensities.
+
+6. Consider residual intensity at the batch level. For RFP, there's not much variation between batches. For GFP, the variation of residual intensities is the largest for plate `18511_18855` and plate `19160_18870`. The plate `18511_18855` was the first one done in imaging, and the plate `19160_18870` contains the most bubbles on the plates based on imaging results.
+
+$~$
+
+### Next steps
+
+1. Assess model convergence conditions: run 10 times on 500 iterations; compare model convergences between different starting points (instead of random time, assign fixed time).
+
+$~$
+
+---
+
+## Load data
+
+$~$
+
+Import combined intensity data (see [combine-intensity-data.R](/project2/gilad/joycehsiao/fucci-seq/code/combine-intensity-data.R)).
+
+```{r, message=FALSE, warning=FALSE, echo = FALSE}
+library(ggplot2)
+library(cellcycleR)
+library(data.table)
+library(dplyr)
+library(cowplot)
+library(wesanderson)
+library(RColorBrewer)
+library(Biobase)
+```
+
+$~$
+
+```{r}
+ints <- readRDS(file="/project2/gilad/joycehsiao/fucci-seq/data/intensity.rds")
+```
+
+```{r, echo = FALSE, eval = TRUE}
+ints$rfp.z <- scale(ints$rfp.median.log10sum, center=TRUE, scale=TRUE)
+ints$gfp.z <- scale(ints$gfp.median.log10sum, center=TRUE, scale=TRUE)
+ints$dapi.z <- scale(ints$dapi.median.log10sum, center=TRUE, scale=TRUE)
+
+ints_tmp <- data.frame(rfp.z=ints$rfp.z,
+                       gfp.z=ints$gfp.z,
+                       dapi.z=ints$dapi.z)
+```
+
+```{r, echo = FALSE, eval = FALSE}
+# cellcycleR analysis
+out <- sin_cell_ordering_class(as.matrix(ints_tmp), 
+                                   celltime_levels=200, num_iter=200)
+saveRDS(out, file = "/project2/gilad/joycehsiao/fucci-seq/output/images-cellcycleR.Rmd/out.rds")
```

Comment:
Since you've added `out.rds` to `output/`, please use a relative path to this file in future updates.